client/python/pyproject.toml

```diff
@@ -29,6 +29,7 @@ dependencies = [
   "python-dateutil>=2.8.2",
   "pyyaml>=5.4",
   "requests>=2.20.0",
+  "packaging",
```

Comment:
Not sure which one, but we probably shouldn't leave it without some min version ?

Reply:
I don't know either, `Version` class is present in this package since v14.0.0 which was a first released version

Reply:
@JDarDagran any thoughts ?

Reply:
Why there should be a minimal version set in the first place?

Reply:
I would rather pin it to some higher version - less chance that something breaks if we use it in some other way and someone else decides to use very low version.